### ISSUE TITLE
Prevent first available day of month from focusing

### ISSUE THREAD

**Issue by dvprz** [airbnb/react-dates#1484]
Hello,

Thank you all for the wonderful library!

So we're working with a team of a11y auditors and one of their suggestion goes against what react-dates does out of the box. When a user tabs to a prev/next button and hits enter after the month transition ends the first available calendar day receives focus. Our consultants want us to keep the focus on the prev/next button. 

I have looked at the source and perhaps I have missed it but I don't see anyway to modify this using the available api. Any ideas?

**Comment by backwardok** [comment]
I agree with this and actually was just looking through issues to see if it was already filed. I think this behavior should be the default, and I would recommend updating the library to change the behavior

**Comment by mmarkelov** [comment]
@backwardok I suppose it is here:
https://github.com/airbnb/react-dates/blob/75bba6b7eb92f57727b45bacdb83221eee6667b6/src/components/CalendarDay.jsx#L58-L69
But I not sure that it won't broke screen readers
